website/docs/guides/docs/sidebar.md

```diff
@@ -391,7 +391,7 @@ module.exports = {
 };
 ```
 
-See it in action in the [Docusaurus Guides pages](../../i18n/i18n-introduction.md).
+See it in action in the [Docusaurus Guides pages](/docs/next/category/guides).
```

Comment:
...is that really future-proof? Or is it only temporary?

Reply:
😅 definitively a shitty temp workaround but better than having the wrong link :p

documented it here: https://github.com/facebook/docusaurus/issues/6041

Reply:
Maybe we should have a hook like `useVersionLink`? 🤔

Reply:
I don't know, if possible I'd prefer to let users keep using regular markdown links than using hooks in their docs